src/files.ts

```diff
@@ -108,10 +108,12 @@ export function parseFileIgnores(path: string): string[] {
     const f = fs.readFileSync(path, { encoding: 'utf8' });
     if (path.includes(DOTSNYK_FILENAME)) {
       // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-      const parsed: { exclude: { code: string[]; global: string[] } } = parseYaml(f);
+      const parsed: { exclude: { code?: string[]; global?: string[] } } = parseYaml(f);
 
       const concatIgnorePath = (p: string) => `${nodePath.dirname(path)}/${p}`;
-      return [...parsed.exclude.code.map(concatIgnorePath), ...parsed.exclude.global.map(concatIgnorePath)];
+      const codeIgnoredPaths = parsed.exclude.code?.map(concatIgnorePath) || [];
```

Comment:
`nit`: Just a personal thing, but I usually use `??` instead of `||`, and use `||` only when I need to take into account `0` in my condition. But, that's my own personal thing so no biggy :)